src/Categories/Category/RigCategory.agda

```diff
@@ -72,6 +79,13 @@ record RigCategory {M⊎ M× : Monoidal C} (S⊎ : Symmetric M⊎)
     ⊕α⇐ = M⊎.associator.to
 
 
+  -- that ann and distrib are natural was missing
```

Comment:
I don't think this comment should be in the code. Maybe change to something like  `naturality conditions for ann and distrib`

Reply:
Good point - fixed.